shapash/explainer/smart_plotter.py

```diff
@@ -1243,11 +1258,11 @@ def features_importance(self,
             total_len = self.explainer.x_pred.shape[0]
             addnote = add_text([addnote,
                                 f"Subset length: {subset_len} ({int(np.round(100 * subset_len / total_len))}%)"],
-                                sep=" - ")
+                               sep=" - ")
         if self.explainer.x_pred.shape[1] >= max_features:
             addnote = add_text([addnote,
                                 f"Total number of features: {int(self.explainer.x_pred.shape[1])}"],
-                                sep=" - ")
+                               sep=" - ")
```

Comment:
there doesn't seem to be the same indentation as the row above

Reply:
True, I will change it

Reply:
ok